config/gulp/images.js

```diff
@@ -4,6 +4,11 @@ var task  = /([\w\d-_]+)\.js$/.exec(__filename)[ 1 ];
 
 gulp.task(task, function (done) {
 
+  dutil.logMessage(task, 'Copying images from docs repo');
+
+  var stream = gulp.src('./img/**/*')
+    .pipe(gulp.dest('assets/img'));
```

Comment:
Just to clarify, will this step change with the assets task updates you are working on are completed?